noir-projects/noir-protocol-circuits/crates/private-kernel-tail/src/main.nr

```diff
@@ -7,7 +7,7 @@ fn main(
     previous_kernel: PrivateKernelDataWithoutPublicInputs,
     previous_kernel_public_inputs: call_data(0) PrivateKernelCircuitPublicInputs,
     include_by_timestamp_upper_bound: u64,
-) -> pub PrivateToRollupKernelCircuitPublicInputs {
+) -> return_data PrivateToRollupKernelCircuitPublicInputs {
```

Comment:
Flagging: We should discuss this change. I think it's fine as long as there's no privacy leak exposing databus commitments. What's not clear to me is whther the data in the databus is genuinely public